### ISSUE TITLE
DeepSpeech support

### ISSUE THREAD

**Issue by ZettaScript** [dessant/buster#306]
The use of remote speech-to-text services has some problems:
* privacy
* unavailability of the service
* need of Facebook or other free or paid accounts
* limitations of the number of requests
* more Internet traffic and latency

Is it conceivable to add support to [DeepSpeech](https://github.com/mozilla/deepspeech), which is free and can be locally hosted?

I don't know if it's mature enough to decode these captchas. I don't even know how to use DeepSpeech.

**Comment by PROPHESSOR** [comment]
+1

Support of DeepSpeech or [other self-hosted speech-to-text translators](https://fosspost.org/open-source-speech-recognition/) would be awesome!

**Comment by merspieler** [comment]
Yes please, lately this wonderful plugin became unusable cause of the persistent API limit..
